domain/cloudimagemetadata/types.go

```diff
@@ -5,6 +5,9 @@ package cloudimagemetadata
 
 import "time"
 
+// CustomSource defines a metadata as "custom" meaning it won't expire
+const CustomSource = "custom"
```

Comment:
Is it possible to be more specific with this description? It doesn't tell me much.